### PR TITLE
Build UI occording to framework

### PR DESCRIPTION
**Pull request by yustarandomname**
This is still in draft because auth was broken during development

### CODE REVIEW

File: app/svelte/src/lib/components/Activity.svelte [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+	$: timeFrom = activity.time?.from.toString({ smallestUnit: 'minute' });
+	$: timeTo = activity.time?.to.toString({ smallestUnit: 'minute' });
+</script>
+
+<Button focused on:click>
+	<div class="flex justify-between items-end mb-2">
+		<span class="text-lg text-bol">{activity.title}</span>
+		<span class="text-sm">
+			{#if activity.isAllDay}
```
Comment (Matei9721) [inline app/svelte/src/lib/components/Activity.svelte:15]: I don't think we can express an "all day" activity in RDF.

File: app/svelte/src/lib/components/input/MultiDate.svelte [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+	export let dates: Temporal.PlainDate[] = [];
+	export let dateStrings: string[] = [];
+	export let label: string = '';
+	export let maxDates: number = 5;
+
+	let newDateString: string = '';
+	let lastDateStrings: string = '';
+
+	function addNewDateString() {
```
Comment (Matei9721) [inline app/svelte/src/lib/components/input/MultiDate.svelte:14]: We also didn't plan to save multiple dates in the event RDF.

File: app/svelte/src/lib/components/Activity.svelte (hunk 1 above)
Comment (Denperidge) [reply app/svelte/src/lib/components/Activity.svelte:15]: We can either do a custom field, or just calculate it using an all-encompassing startDate/endDate!

Comment (Denperidge) [reply app/svelte/src/lib/components/Activity.svelte:15]: Boolean is a valid data type, so honestly the custom field will be easier https://schema.org/Boolean

**Comment by yustarandomname** [comment]
closes #51

**Pull request merged**